### PR TITLE
alternatives comparison update

### PR DESCRIPTION
**Pull request by dustinblack**
## Changes introduced with this PR

Adjusting the language of the alternative comparisons per #52 

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dustinblack** [comment]
@jtaleric @kdelee -- Please take a look at this initial draft change in the alternatives documentation for Ansible only. I've tried to take a more matter-of-fact FAQ approach, as originally intended for this documentation, and I've eliminated the "better than" and "worse than" language.

**Review by jtaleric — APPROVED** [review]
/lgtm @dustinblack - 

I think the updated language is better.

**Comment by dustinblack** [comment]
@matburt FYI

**Comment by dustinblack** [comment]
PR is now live and ready for review. I'd love to get outside reviews from @jtaleric @kdelee @psuriset and @matburt for the updates related to Ansible.

Also pinging @johnaohara and @stalep for review and any updates to the qDup comparison section.

Thanks!

**Review by jdowni000 — APPROVED** [review]
Excellent job rewording this

**Review by Harshith-umesh — APPROVED** [review]
LGTM

File: docs/arcaflow/index.md [hunk 1]
```diff
@@ -21,149 +22,138 @@ You can use Arcaflow for many things. We use it for:
...
+    - They can both run things on remote systems out of the box.
 
-    **What it does worse than Arcaflow**
+    **How is qDup different?**
 
-    - It does not have a typing system and cannot guarantee type-safety or validation of data
-    - It doesn't integrate with container engines natively
+    - qDup has advanced controls for creating loops, signaling, waiting for events, etc.
+    - qDup makes it very simple to run scripts in a parallelized way.
+    - qDup runs comands over SSH by design.
```
Comment (johnaohara) [inline docs/arcaflow/index.md:157]: Hi @dustinblack. qDup currently runs over SSH, locally (i.e on a local machine without the SSH requirement), or integrates with podman. There are plans to support docker/ocp and k8's in the future. https://github.com/Hyperfoil/qDup/blob/master/docs/reference/hosts.adoc

File: docs/arcaflow/index.md [hunk 2]
```diff
@@ -21,149 +22,138 @@ You can use Arcaflow for many things. We use it for:
...
+    - qDup is written in Java.
+    - qDup does not integrate with container engines natively.
```
[the remark this answers is not in the document]
Comment (dustinblack) [reply docs/arcaflow/index.md]: Thanks @johnaohara! I've update the comparison with this new info.

**Pull request merged**